app/assets/stylesheets/semantic-ui/globals/_variables.scss

```diff
@@ -1,4 +1,53 @@
-$import-google-fonts: true !default;
-$font-url: 'https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic&subset=latin,latin-ext' !default;
-$font-name: 'Lato' !default;
-$font-family: $font-name, 'Helvetica Neue', Arial, Helvetica, sans-serif !default;
+// Fonts
+$font-url: '';
+$import-google-fonts: false;
+$font-name: 'proxima-nova' !default;
+$font-family: $font-name, Arial, Helvetica, sans-serif !default;
+
+// Global
+$white: #FFFFFF;
+$text-shadow: 0 1px 3px rgba(0,0,0,.5);
+$box-shadow: 0 3px 6px rgba(0,0,0,.5);
+$box-radius: 0.485rem;
+
+// Button Effects
+$button-shadow: 0 3px 6px rgba(0,0,0,.25);
+$button-padding: 0.875em 1.5em 0.875em;
+$button-radius: 0.425rem;
+$button-border: 1px solid rgba(255, 255, 255, 0.15);
+$button-border-hover: 1px solid rgba(255, 255, 255, 0.4);
+
+// Input Styling
+$input-bg: rgba(0, 0, 0, 0.4);
+$input-padding: 0.685em 1em 0.685em;
+$input-color-inverted-placeholder: rgba(0,0,0,.4);
+
+//Box Gradients
+$box-gray-gradient: linear-gradient(#FFFFFF, #DEDEDE);
+
+// Button Gradients
+$button-primary-gradient: linear-gradient(#4DB1D8, #1274B2);
+$button-primary-gradient-hover: linear-gradient(#6ECBEF, #177BBA);
+$button-secondary-gradient: linear-gradient(#838890, #5F636A);
+$button-secondary-gradient-hover: linear-gradient(#9CA1A8, #696D74);
+$button-positive-gradient: linear-gradient(#6AF04A, #1B9A29);
+$button-positive-gradient-hover: linear-gradient(#8BFF6F, #21AD31);
+$button-negative-gradient: linear-gradient(#FC6C6C, #CC3030);
+$button-negative-gradient-hover: linear-gradient(#FF8D8D, #E04545);
+
+// Text Color
+$title-color-inverted: rgba(0,0,0,.85);
+$text-color: rgba(255,255,255,.6);
+$text-color-inverted: rgba(0,0,0,.6);
+$text-error: #FF7B7B;
+$text-color-disabled: rgba(255, 255, 255, 0.4);
+
+// Dropdown Styling
+$dropdown-hover-bg: #ebf7ff;
+
+// Divider Styling
+$divider-color: 1px solid rgba(255, 255, 255, 0.05);
+$divider-color-inverted: 1px solid rgba(0, 0, 0, 0.05);
```

Comment:
I think `-color` is misleading here since it's not a color, I would prefer `-border` or having no prefix